tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -1678,5 +1678,19 @@ describe("Service Form Modal", function() {
         .siblings()
         .should("to.have.length", 2);
     });
+
+    it("Should show network form when clicking on Network Configuration Edit", function() {
+      cy.get(".menu-tabbed-item").contains("Networking").click();
+      cy.get(".menu-tabbed-view .button.button-primary-link").first().click();
+      cy.get('input[name="containers.0.endpoints.0.name"]').type("test");
+      // Click review and run
+      cy
+        .get(".modal-full-screen-actions")
+        .contains("button", "Review & Run")
+        .click();
+      // Click edit to view form
+      cy.get("a.button.button-link").eq(-1).click({ force: true });
```

Comment:
Yes, we will go with force then, since:

1) The invoke("show") actually uses jQuery's show method :(
2) We'll have to access the DOM and either change the visibility to visible or create a custom mouseover event and dispatch that event.

We don't use jQuery and the second method is sketchy as we move away from the abstraction of Cypress.